### ISSUE TITLE
optionally pre-fill minibuffer prompt with thing-at-point

### ISSUE THREAD

**Issue by fgeller** [syohex/emacs-anzu#77]
It would be great if I could use a variant of `anzu-query-replace` with the current thing-at-point pre-filled into the prompt. `anzu-query-replace-at-cursor` fixes the replacement candidate and I'd like to be able to edit the candidate before I start replacing. Would that be possible?

**Comment by syohex** [comment]
No, current implementation does not have such feature. Is it enough to use copy/paste and replace command(`anzu-query-replace` etc) in such case ?
